### PR TITLE
Update base image

### PR DESCRIPTION
**Pull request by esron**
[APPSRE-10994](https://issues.redhat.com/browse/APPSRE-10994)

This updates the base image to tag [1-201.1729679484](https://catalog.redhat.com/software/containers/ubi8/python-39/6065b24eb92fbda3a4c65d8f).

Also I'm fixing the exposed port so `make test-data` doesn't fail.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
-FROM        registry.access.redhat.com/ubi8/python-39:1-83
-
-WORKDIR     /dashdotdb
+FROM        registry.access.redhat.com/ubi8/python-39:1-201.1729679484
```
Comment (cubismod) [inline Dockerfile:1]: Can you try updating to Ubi9 as well?

Comment (esron) [reply Dockerfile:1]: For sure.

**Review by cubismod — APPROVED** [review]
LGTM

**Pull request merged**